### PR TITLE
Update to use Cloud Scheduler

### PR DESCRIPTION
**Pull request by jpatokal**
The functionality of the App Engine app in this repo is now implemented as product by [Cloud Scheduler](https://cloud.google.com/scheduler/).   This PR removes the App Engine portion and replaces it with step-by-step instructions for how to integrate Cloud Scheduler with the Compute Engine utility.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,39 +1,30 @@
-# Sample: Reliable Task Scheduling on Google Compute Engine
+# Sample: Reliable Task Scheduling on Google Compute Engine with Cloud Scheduler
+
 In distributed systems, such as a network of Google Compute Engine
 instances, it is challenging to reliably schedule tasks because any individual
 instance may become unavailable due to autoscaling or network partitioning.
 
-Google AppEngine provides a Cron service. Using this service for scheduling and
+Google Cloud Platform provides a managed Cloud Scheduler service. Using this service for scheduling and
```
Comment (ptone) [inline readme.md]: link cloud scheduler?

File: readme.md [hunk 2]
```diff
@@ -149,43 +115,46 @@ that you created in **Prerequisites**.
...
-1. Include the Python API client in your App Engine application.
+1.  Create the Pub/Sub topic that you will push messages to.
 
-        $ pip install -t gae/lib/ google-api-python-client
+        $ gcloud pubsub topics create test
 
-    Note: if you get an error and used Homebrew to install Python on OS X,
-    see [this fix](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Homebrew-and-Python.md#note-on-pip-install---user).
+The topic is now listed under `gcloud pubsub topics list`.  You can also see the topic
+(after about a minute) in the console:
```
Comment (ptone) [inline readme.md]: I think this create is now much faster than 1-minute

**Review by ptone — APPROVED** [review]
Comments are minor/optional, LGTM

File: readme.md (hunk 1 above)
Comment (jpatokal) [reply readme.md]: Linked.

File: readme.md (hunk 2 above)
Comment (jpatokal) [reply readme.md]: Ta, removed.

**Pull request merged**